File: semester2/sysprog/sypr-BspProgramme/einstieg.cpp
//einstieg.cpp

#include <iostream>
#include <string>

int main()
{
    std::cout << "Anzahl bisher geschriebener C++-Programme eingeben: ";
    int anzahl;
    std::cin >> anzahl;

    std::cout << "Vorname eingeben: ";
    std::string vorname;
    std::cin >> vorname;

    std::cout << vorname << "s " << anzahl + 1 << ". C++-Programm funktioniert!\n";

    //return 0; wird automatisch ergaenzt
}

File: semester1/prog1/aufgabe1/aufgabe1/prog1-uebungen/einstieg/Einstieg.java
//Einstieg.java
package einstieg;

import java.util.Scanner;

/** 
* Einstieg ist ein kleines Java-Programm.
* Es verwendet die Klassen Scanner und System aus der Java-Bibliothek.
* @author TODO: Name eintragen und TODO inklusive Doppelpunkt löschen
* @version TODO: Datum eintragen und TODO inklusive Doppelpunkt löschen
*/
public final class Einstieg {
    private Einstieg() { }

    private static final Scanner EINGABE = new Scanner(System.in);

    /** 
    * Die Klassenmethode main ist der Startpunkt des Programms.
    * main verwendet die Methoden print und printf zum Ausgeben von Text
    * sowie die Methoden next und nextInt zum EInlesen von Text
    * und einer ganzen Zahl.
    * @param args wird nicht verwendet
    */
    public static void main(String[] args) {
        System.out.print("Vorname: ");
        String vorname = EINGABE.next();

        System.out.print("Anzahl bisher geschriebener Java-Programme: ");
        int anzahl = EINGABE.nextInt();

        System.out.printf("%ss %d. Java-Programm funktioniert!%n", vorname, anzahl + 1);
    }
}